providers/attio/parse.go

```diff
@@ -0,0 +1,10 @@
+// nolint
+package attio
+
+import (
+	"github.com/spyzhov/ajson"
+)
+
+var dummyNextPageFunc = func(*ajson.Node) (string, error) {
```

Comment:
Can you move this over to `read`? Would be good to have lesser files whenever possible

Reply:
This function should belong in common. I can see other connectors using it. As a default behaviour -- no pagination.

Reply:
I removed dummyNextPageFunc function because some object in Attio connector has pagination. Instead of this function, I created new function makeNextRecordsURL function in attio/parse.go file.